README.md

```diff
@@ -1 +1,18 @@
-# training-ml-platforms
\ No newline at end of file
+# ML on Discovery
+This repository contains material for ML on Discovery
+training modules.
+
+## What's in each training sub-folder?
+* Training PowerPoint slides
```

Comment:
I thought the powerpoints were only on sharepoint. Should we be putting them in the repos also?

Reply:
I just meant to say that they are part of the training material, not pertaining to the repo. Maybe I'll revise this statement after the training, if it's confusing.